views/administration/users_skipregistration.hbs

```diff
@@ -57,7 +57,7 @@
 
                     {{#unless editTeacher}}
                     <div class="col-md-6">
-                        {{$t "administration.global.text.forStudentsUnder" (dict "minAge" CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS )}}<br />
+                        {{$t "administration.global.text.forStudentsUnder" (dict "minAge" age )}}<br />
```

Comment:
please use same placeholder on the text, so it is more clear if someone taking a look on it

Reply:
age is already used in other places for the same var, so while it may make sense to unify this but is out of scope for this tickt.